nifi-bootstrap/pom.xml

```diff
@@ -41,5 +41,12 @@
 			<groupId>org.apache.nifi</groupId>
 			<artifactId>nifi-expression-language</artifactId>
 		</dependency>
+		<dependency>
+          <groupId>com.sun</groupId>
+          <artifactId>tools</artifactId>
+          <version>1.7.0+</version>
+          <scope>system</scope>
+          <systemPath>${java.home}/../lib/tools.jar</systemPath>
```

Comment:
Actually, we can remove that  declaration in pom. tools.jar/classes.jar has only runtime dependency.